### PR TITLE
Format node examples with standard

### PR DESCRIPTION
**Pull request by nickcolley**
## What problem does the pull request solve?
Just to scratch an itch based on the style of the code, so it matches what we do for other projects: https://gds-way.cloudapps.digital/manuals/programming-languages/nodejs/#source-formatting-and-linting

This turned into a big pull request and through doing this I have found some errors in the JavaScript but mostly this is stylistic stuff.

Happy to close this or do less, let me know what you want to do.

View changes with whitespaces removed from the diff:
https://github.com/alphagov/notifications-node-client/pull/63/files?w=1

## Checklist
- [x] I’ve used the pull request template
- [x] I’ve written unit tests for these changes
- [x] I’ve updated the documentation in
  - `DOCUMENTATION.md`
  - `CHANGELOG.md`
- [ ] I’ve bumped the version number in
  - `package.json`
- [ ] I've added new environment variables in
  - `CONTRIBUTING.md`
  - `notifications-node-client/scripts/generate_docker_env.sh`

### CODE REVIEW

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ If a template has placeholder fields for personalised information such as name o
 A unique identifier you create. This reference identifies a single unique notification or a batch of notifications. If you do not have a reference, you must pass in an empty string or `null`. For example:
 
 ```
-"your_reference_here";
+"your_reference_here"
```
Comment (quis) [inline DOCUMENTATION.md]: Should this be using single quotes too?

Comment (nickcolley) [reply DOCUMENTATION.md]: Yeah likely, I think since this is not using ```javascript standard-markdown doesnt lint it?

File: README.md [hunk 2]
```diff
@@ -8,3 +8,20 @@ Useful links:
 - [NPM package](https://www.npmjs.com/package/notifications-node-client)
 - [Changelog](https://github.com/alphagov/notifications-node-client/blob/master/CHANGELOG.md)
 - [Contributing to this client](https://github.com/alphagov/notifications-node-client/blob/master/CONTRIBUTING.md)
+
+## Testing JavaScript examples in Markdown
```
Comment (quis) [inline README.md]: Should look at how to make this run as part of continuous integration.

Comment (nickcolley) [reply README.md]: I have added this to the 'npm test' command which runs automatically.

Comment (quis) [reply README.md]: Yeah, noticed straight after I posted that.

***

Also, this section should go in `CONTRIBUTING.md` not `README.md` (we separate out guidance that isn’t relevant to people consuming the client by putting it in there).

**Comment by nickcolley** [comment]
Think I'm going to have a fresh look at this tomorrow, and properly look over everything.

**Comment by quis** [comment]
@nickcolley is there any help you need from us to get this in a mergeable state?

**Pull request merged**